_problems/removing-cross-layer-links.md

```diff
@@ -3,7 +3,7 @@ layout: problem
 title:  "Removing cross-layer links"
 tags: lightning privacy
 status: open
-maintainer: LLFourn
+maintainer: LLFourn, ariard
```

Comment:
Note to self: I will have to change the template to make multiple authors work.

Reply:
Dropping myself for now.